### ISSUE TITLE
This addon entirely stopped working between Sep 2nd and Sep 3rd 2024 (hints do not appear)

### ISSUE THREAD

**Issue by langfield** [lydell/LinkHints#97]
### Screen shot
There's nothing to show. No hints are displayed, though the counter badge does appear when you attempt to initiate a click.

### Description
Hints do not appear. I fixed this problem by downgrading to `1.3.1`.

### Debug info
<details>
<summary>Debug info</summary>

```json
{
  "version": "1.3.2",
  "browser": "firefox",
  "userAgent": "Mozilla/5.0 (X11; Linux aarch64; rv:109.0) Gecko/20100101 Firefox/115.0",
  "browserInfo": {
    "name": "Firefox",
    "vendor": "Mozilla",
    "version": "115.6.0",
    "buildID": "20231211164624"
  },
  "platformInfo": {
    "os": "linux",
    "arch": "aarch64"
  },
  "storage.sync": {},
  "storage.local": {
    "tutorialShown": true
  },
  "language": "en-US"
}
```

</details>

**Comment by lydell** [comment]
Firefox 115 is over a year old. Is it possible for you to test with a newer Firefox?

**Comment by lydell** [comment]
I figured it out. I have released 1.3.3 (available on addons.mozilla.org soon) with support for Firefox <125.
